pkg/prompt/prompt.go

```diff
@@ -84,12 +84,14 @@ func Kind(provider providerregistrysdk.ProviderDetail) (string, error) {
 	}
 	return selectedProviderKind, nil
 }
-func Provider(ctx context.Context, registryClient *registryclient.Client) (*providerregistrysdk.ProviderDetail, error) {
+func Provider(ctx context.Context, registryClient *registryclient.Client, shouldShowHiddenProviders bool) (*providerregistrysdk.ProviderDetail, error) {
```

Comment:
```suggestion
func Provider(ctx context.Context, registryClient *registryclient.Client, showHiddenProviders bool) (*providerregistrysdk.ProviderDetail, error) {
```
Minor nitpick on the naming here